documentation/bolt_new_features.md

```diff
@@ -0,0 +1,438 @@
+---
+author: Jean Bond <jean@puppet.com\>
+---
+
+# New features
+
+New features added to Bolt in the 1.x release series.
```

Comment:
Is this page accessible? I think it was probably orphaned when we switched to a changelog